src/status_im/init/core.cljs

```diff
@@ -20,7 +21,8 @@
               :keycard (dissoc keycard :secrets)
               :supported-biometric-auth supported-biometric-auth
               :app-active-since (or app-active-since now)
-              :multiaccounts/loading true)})
+              :multiaccounts/loading true)
+   :dispatch [::anon-metrics/fetch-opt-in-screen-displayed?]})
```

Comment:
could we move this outside ? because `initialize-app-db` doesn't suppose to do any side effects

Reply:
moved it to `initialize-multiaccounts`